wire/wire.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package basictracer_go.wire;
+option go_package = "wire";
+
+message TracerState {
```

Comment:
Per https://github.com/opentracing/opentracing.github.io/issues/73#issuecomment-193499918, the semantics of the split between "tracer state" and "baggage" is up in the air (and – unless someone objects – I suspect they will be unified). Not sure if we should (a) wait for https://github.com/opentracing/opentracing.github.io/issues/73 to resolve before merging this or (b) go with what you have here with the understanding that it's going to change imminently.